index.html

```diff
@@ -4,45 +4,86 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>The FlowerPot</title>
-</head>
+    <!-- defer attribute runs the script after the HTML has rendered -->
+    <script src="assets/scripts/script.js" defer></script>
+    <link rel="stylesheet" href="assets/css/reset.css">
+    <link rel="stylesheet" href="assets/css/style.css">

```

Comment:
Note: I believe best practice is to always have your CSS stylesheet linked at the bottom. To ensure your styling is applied over any other styles you might be linking in.

Reply:
Best practice to link the style sheet at the bottom of the head element, correct.
More info can be found here:https://stackoverflow.com/questions/18392418/html-css-put-link-tag-outside-of-the-head#:~:text=7%20Answers&text=Style%20sheets%20are%20linked%20in,document%20from%20the%20top%20again 

the script can be linked at the bottom of the body and this makes it run after the rendering has taken place. However the 'defer' attribute does the same thing, forces the script to wait till rendering has finished before executing. 
I used defer to keep all the linked external files together.

Any futher questions, dont hesitate.

Reply:
I was unaware of the defer attribute! That's great to know about, thanks for explaining.